app/component/search.scss

```diff
@@ -154,6 +159,107 @@ $search-header-height: 45px;
   top: -17px;
 }
 
+.onetab-search-modal-container.bp-large {
```

Comment:
There seems to be quite much overlap between this and https://github.com/HSLdevcom/digitransit-ui/pull/1029/files#diff-479d8103fbc0dc78db0a4c3217882501R72. Could we refactor somehow to get rid of the duplication?